### ISSUE TITLE
Syncthing has stopped / Syncthing keeps stopping

### ISSUE THREAD

**Issue by luontokoodaus** [syncthing/syncthing#5080]
Device: HTC U11+
Android: 8.0.0
Syncthing: Latest Android

Creating a folder and sharing it works. Receiving access to a folder does not work. When selecting a directory to have the files/folders from another device saved onto my HTC U11+ the app crashes 100% of the time, as soon as I choose a directory on my phone's internal memory. Choosing a directory on the external SD card works, but then syncthing can't write any files to this directory. So the folder stays empty and the status "Out of Sync" never changes.

In short. I can share files and folders, but I cannot receive links to files/folders without the app crashing 100% of the time.

I have tried:

1. Reboot
2. Uninstall / Reinstall
3. Clear app data/cache
4. Different folder names
5. Much more

**Comment by ranfdev** [comment]
Same thing here...
I receive a notification saying:
> Device "xxx" wants to share folder "yyy"...
I click on the notification, i set the folder and the other options, click to save and the app crashes. The changes are not saved and i cant add any folder.
Android: 8.1.0
Syncthing: 0.14.49

**Comment by luontokoodaus** [comment]
Yep this is exactly the same problem I am having with the latest version. Didn't have it with another version I used months ago.

**Comment by calmh** [comment]
You guys probably want the syncthing-android issue tracker at https://github.com/syncthing/syncthing-android/issues
